Fix Bugzilla bug list table resizing

jdatapple opened the pull request:
#### d6aabc0803dc75b76576aa86544ec7fa43f57504
<pre>
Fix Bugzilla bug list table resizing
<a href="https://bugs.webkit.org/show_bug.cgi?id=283369">https://bugs.webkit.org/show_bug.cgi?id=283369</a>

Reviewed by Alexey Proskuryakov.

This set of changes makes numerous small tweaks to column hiding and
column sizing in order to maximize the visibility of critical bug
information when resizing the viewport.

* Websites/bugs.webkit.org/skins/custom/buglist.css:
(.bz_buglist):
(.bz_buglist .bz_id_column):
(.bz_buglist .bz_component_column):
(.bz_buglist .bz_reporter_column,):
(.bz_buglist .bz_changeddate_column):
(@media only screen and (max-width: 1500px) .bz_buglist .bz_product_column):
(@media only screen and (max-width: 1300px) .bz_buglist .bz_assigned_to_column):
(@media only screen and (max-width: 900px) .bz_buglist .bz_component_column):
(@media only screen and (max-width: 730px) .bz_buglist .bz_bug_status_column):
(@media only screen and (max-width: 600px) .bz_buglist .bz_short_desc_column):
(.bz_buglist_header th:nth-child(6),): Deleted.
(.bz_buglist .bz_assigned_to_column): Deleted.
(@media only screen and (max-width: 1500px) .bz_buglist_header th:nth-child(2),): Deleted.
(@media only screen and (max-width: 1300px) .bz_buglist_header th:nth-child(4),): Deleted.
(@media only screen and (max-width: 900px) .bz_buglist_header th:nth-child(3),): Deleted.
(@media only screen and (max-width: 730px) .bz_buglist_header th:nth-child(5),): Deleted.
(@media only screen and (max-width: 428px) .bz_buglist .bz_id_column): Deleted.
(@media only screen and (max-width: 428px) .bz_buglist .bz_short_desc_column): Deleted.
* Websites/bugs.webkit.org/template/en/custom/list/table.html.tmpl:

Canonical link: <a href="https://commits.webkit.org/286860@main">https://commits.webkit.org/286860@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f7c60d79f11eec2034e7dc8c2cf47d06bf5529bf

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/77163 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/56198 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/30078 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/81723 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/28445 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/65346 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/4494 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/60454 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/18515 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/80230 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/50418 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/66233 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/40757 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/47820 "Passed tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/26768 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/68935 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/24066 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/83150 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/4543 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/3056 "Found 1 new test failure: webrtc/vp8-then-h264.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/68735 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/4699 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/66206 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/67990 "Found 1 new API test failure: /TestWebKit:WebKit.GetInjectedBundleInitializationUserDataCallback (failure)") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/11977 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/10069 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11968 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/4490 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/4509 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/7944 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/6268 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (approved):
rs=me. It would be good to clean up autogenerated lines in the commit messages, most of those seem unhelpful.

jdatapple committed:
Fix Bugzilla bug list table resizing
https://bugs.webkit.org/show_bug.cgi?id=283369

Reviewed by Alexey Proskuryakov.

This set of changes makes numerous small tweaks to column hiding and
column sizing in order to maximize the visibility of critical bug
information when resizing the viewport.

* Websites/bugs.webkit.org/skins/custom/buglist.css:
(.bz_buglist):
(.bz_buglist .bz_id_column):
(.bz_buglist .bz_component_column):
(.bz_buglist .bz_reporter_column,):
(.bz_buglist .bz_changeddate_column):
(@media only screen and (max-width: 1500px) .bz_buglist .bz_product_column):
(@media only screen and (max-width: 1300px) .bz_buglist .bz_assigned_to_column):
(@media only screen and (max-width: 900px) .bz_buglist .bz_component_column):
(@media only screen and (max-width: 730px) .bz_buglist .bz_bug_status_column):
(@media only screen and (max-width: 600px) .bz_buglist .bz_short_desc_column):
(.bz_buglist_header th:nth-child(6),): Deleted.
(.bz_buglist .bz_assigned_to_column): Deleted.
(@media only screen and (max-width: 1500px) .bz_buglist_header th:nth-child(2),): Deleted.
(@media only screen and (max-width: 1300px) .bz_buglist_header th:nth-child(4),): Deleted.
(@media only screen and (max-width: 900px) .bz_buglist_header th:nth-child(3),): Deleted.
(@media only screen and (max-width: 730px) .bz_buglist_header th:nth-child(5),): Deleted.
(@media only screen and (max-width: 428px) .bz_buglist .bz_id_column): Deleted.
(@media only screen and (max-width: 428px) .bz_buglist .bz_short_desc_column): Deleted.
* Websites/bugs.webkit.org/template/en/custom/list/table.html.tmpl:

Canonical link: https://commits.webkit.org/286860@main
